train.py

```diff
@@ -84,14 +84,14 @@ def train_net(net,
 1. When do BasicDataset function(in src.unet_.dataset.py), image shpae become 5 dimension (2, 19, 681, 1086, 12) (batch size, yr, y, x, band).
 But the dimension in the model is 4 dimension (64, 3, 3, 3);
 
-2. The input size in UNet model should be (N, Cin, H, W): N is N is a batch size, CC denotes a number of channels, 
+2. The input size in UNet model should be (N, Cin, H, W): N is a batch size, CC denotes a number of channels, 
```

Comment:
For the training and preduction I suggest splitting up the full image into tiles (e.g. of size 128 x 128 or 256x 256 or so) - In the ArcGIS demo they use 400 x 400 
https://developers.arcgis.com/python/sample-notebooks/land-cover-classification-using-unet/